src/Cedar/SM.c

```diff
@@ -18365,6 +18365,7 @@ void SmServerDlgInit(HWND hWnd, SM_SERVER *p)
 void SmServerDlgRefresh(HWND hWnd, SM_SERVER *p)
 {
 	RPC_ENUM_HUB t;
+	RPC_LISTENER_LIST t2;

```

Comment:
Why is this now called `t2` instead of `t`?

Reply:
This line moved up from orignal version. 
 Line 18367 has defined a param t, so this param shouldn't be t too, so I rename it to t2.

Reply:
Right.

I think that the variables should have a better name, such as `RpcEnumHub` and `RpcListenerList`.

Reply:
Yes, you are right, you can rename this param freely, and make a new PR. 

The param names in this project are not always self-declared,  for examples, a lot of single char param, such as "o"  ,"a", "b", "c", "p","s", "t"  there.  What's more , they are a lot of MACRO used to declare/define functions, such as rpc-processing code, it makes more diffcult to do smart-param-rename. But nobody trys to rename all of them, the work is too huge, on the other hand it becomes a new code for authors/ users  who are familiar with the current style.  I don't  know how to deal with this situation.

Reply:
I would leave the names as they are, for now.

We could create an issue in order to discuss about it and decide what to do in future.

Reply:
Great idea. I hope to join the discuss.

Reply:
Feel free to create the issue, if you want to.